package.json

```diff
@@ -76,26 +76,26 @@
     "copyfiles": "^2.4.1",
     "cross-env": "^7.0.3",
     "delay": "^5.0.0",
-    "esbuild": "^0.14.1",
+    "esbuild": "^0.14.8",
```

Comment:
I think we want to be careful about this one because of https://github.com/sequelize/sequelize/issues/13791#issuecomment-1001246172

Please hardcode 0.14.1 instead (without the ^)

Reply:
Yeah, I was using this PR to check if the tests would fail. Also cc @ephys in this

Reply:
👍